auth/ssh.php

```diff
@@ -94,7 +94,21 @@ public function connect() {
   public function send_command($command) {
```

Comment:
Maybe modify this function to pass a reference to the `Router` object. And then create an abstract function in the `Router` class to be defined for each subclass, then to be called in place of the `if` statement you introduced.

Reply:
that sounds good, I'l try to implement this...